lib/core/middleware/nuxt.js

```diff
@@ -68,9 +68,13 @@ module.exports = async function nuxtMiddleware(req, res, next) {
     }
 
     if (this.options.render.csp) {
+      let allowSources = cspScriptSrcHashes
```

Comment:
`let allowedSources = cspScriptSrcHashes.concat(this.options.render.csp.allowDomains)`